sycl/include/CL/sycl/detail/pi.h

```diff
@@ -294,6 +294,7 @@ typedef enum {
       CL_DEVICE_CROSS_DEVICE_SHARED_MEM_CAPABILITIES_INTEL,
   PI_DEVICE_INFO_USM_SYSTEM_SHARED_SUPPORT =
       CL_DEVICE_SHARED_SYSTEM_MEM_CAPABILITIES_INTEL,
+  PI_DEVICE_INFO_BACKEND_VERSION = 0x1f000,
```

Comment:
I am okay with the selected value, but would you mind placing it after `PI_DEVICE_INFO_GPU_HW_THREADS_PER_EU` just so we have values in increasing order to avoid people picking taken values in the future? If you want, it should be fine to also take `0x10113` as it is a non-extension PI query without a direct OpenCL equivalent.

Reply:
I wanted to leave some space for potential new values in OpenCL, thats why the skip to 0x1f000.

Reply:
We start custom values at `0x10020` so we have set a lower bound on this already, so `0x10113` shouldn't be the first to cause problems if it ever happens.

Reply:
On second thought, these may be OpenCL extensions not yet reflected. Maybe this should be a `PI_EXT_ONEAPI_DEVICE_INFO_` instead?

Reply:
No, this is part of core SYCL spec, not an extension. I don't think ti matters for naming of this property, whether it is core/experimental/not present in OpenCL. Only potentially for what value we give it.

Reply:
Right you are. Looking at the PI backends I believe `0x10113` should be the value of this.

Reply:
done